### PR TITLE
[node-feature-discovery] Initial version

### PR DESCRIPTION
**Pull request by Juggels**
#### Special notes for your reviewer:
Solves #233 

3 open questions for now:

* Currently the volumes and volumeMounts are static, based on the example provided by the source repo. I'm thinking it may be beneficial to allow additional mounts via the `values.yaml`?

* Not 100% sure what the best approach would be for location of `service` in the `values.yaml`, as the service is basically only used by the master. Maybe it should be better put there?

* Also not completely completely sure about if I should put the component in the filenames of the daemonset/deployment.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/node-feature-discovery/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# deCONZ helm chart
```
Comment (billimek) [inline charts/node-feature-discovery/README.md:1]: This should probably be `node-feature-discovery`

Comment (Juggels) [reply charts/node-feature-discovery/README.md]: You're probably right, fixed in latest commit 😉

**Comment by Juggels** [comment]
Running this chart now, and very happy that this repo put me on to it, it saves me from manually tagging and updating my nodes' USB dongles and such 😄  

I'm still somewhat struggling with the fact that the application has a master and a worker component and how to best reflect that in the chart/values, as you can see in the initial questions... If you have some time available maybe you could share your views on that?

**Comment by billimek** [comment]
* Regarding the service: I believe it is fine the way you have it structured.
* Regarding the possibility of extra volume mounts: let's tackle that if it becomes needed in a future change.

Thanks so much for adding this chart, node-feature-discovery is such a nice alternative to manually labeling nodes, like you say.

**Pull request merged**